hypothesis_sqlalchemy/records.py

```diff
@@ -166,6 +167,11 @@ def uuid_type_values_factory(_: UUID) -> Strategy:
     return strategies.uuids()
 
 
+@from_column_type.register(LargeBinary)
+def bytes_type_values_factory(bytes_type: LargeBinary) -> Strategy:
+    return binary(min_size=bytes_type.length, max_size=bytes_type.length)
+
+
 factories_by_sql_types = {
     String: string_type_values_factory,
     Enum: enum_type_values_factory,
```

Comment:
can you please remove this `factories_by_sql_types` structure? it is a legacy artifact